extensions/npm/main.ts

```diff
@@ -27,21 +27,21 @@ class FileBackup {
       } else {
         await Deno.remove(this.fileName);
       }
-    } catch (e) {}
+    } catch (_e) { /* Do nothing */ }
   }
 }
 
 // Find project root directory.
-async function findRoot(manifest: string): string | undefined {
+async function findRoot(manifest: string): Promise<string | undefined> {
```

Comment:
Weird that this wasn't caught already by `prettier` or at runtime by `deno` itself. I definitely remember `deno` complaining about an `async` function that did not return `Promise` in its signature.

Reply:
Type information is erased at runtime and `prettier` doesn't do type checking. Even in this branch, type checking is never performed. This was only caught because of my editor setup. I will bring that up in a separate issue and PR.